server/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
Yes, its best practice for courtesy. Also, its auto-gen from IBM since I used their starter kit, I am keeping it there for acknowledgment. It will also be better for us in the future whenever we are referring back to this project for works or anything. I will edit/move them to the outer folder in the future.

Did you know even SCE has one? (Corev4)
![image](https://user-images.githubusercontent.com/18486562/88122950-b7e86500-cb7e-11ea-841e-870638f009ba.png)

Reply:
there's a lot of sce secrets we don't know about

Reply:
README is extremely important for documentation

License and conducts are for courtesy purposes

At the end of the day, we want to present this more than just a "student project"

Reply:
Yes, I have seen the SCE one, but wasn't sure if we needed it for this. Thanks for the explanation. :))